drivers/gles3/shaders/scene.glsl

```diff
@@ -129,7 +129,7 @@ layout(std140) uniform SceneData { // ubo:2
 
 	mediump float ambient_color_sky_mix;
 	bool material_uv2_mode;
-	float pad2;
+	float emissive_exposure_normalization;
```

Comment:
Good catch here, these should have been renamed when Physical Light Units were added